app/src/main/java/woowacourse/payments/ui/model/CardCompanyUiModel.kt

```
@@ -1,87 +1,80 @@
+        @ColorRes val color: Int,
         @DrawableRes val logo: Int,
         val company: CardCompany,
     ) : CardCompanyUiModel
 }
 
 fun CardCompany.toUiModel(): CardCompanyUiModel.SelectCardCompany = cardCompanyMap.getValue(this)
-fun CardCompanyUiModel.toDomain(): CardCompany? =
+fun CardCompanyUiModel.toDomain(): CardCompany =
     when (this) {
-        is CardCompanyUiModel.Default -> null
+        is CardCompanyUiModel.Default -> throw IllegalArgumentException()
```

detekt (1.23.8)

[warning] A call to the default constructor of an exception was detected. Instead one of the constructor overloads should be called. This allows to provide more meaningful exceptions.

(detekt.exceptions.ThrowingExceptionsWithoutMessageOrCause)